tutorials/io/saving_games.rst

```diff
@@ -267,7 +267,7 @@ load function:
         while (saveGame.GetPosition() < save_game.GetLen())
         {
             // Get the saved dictionary from the next line in the save file
-            var nodeData = (Godot.Collections.Dictionary<string, object>)JSON.Parse(saveGame.GetLine()).Result;
+            var nodeData = new Godot.Collections.Dictionary<string, object>(JSON.Parse(saveGame.GetLine()).Result as Godot.Collections.Dictionary);
```

Comment:
It's a good practice to never use the `as` operator unless you will check if the conversion succeeded. It's better to do a normal cast and get a cast exception if it fails instead of a null exception somewhere after.

Reply:
100% agree good point!

Updated